src/app/auth.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+interface MyData {
```

Comment:
Please use descriptive names. If I look into object of type MyData, I have no idea what this.